app/proguard-rules.pro

```diff
@@ -0,0 +1,2 @@
+-keep class com.artemzin.qualitymatters.models.** { *; }
+-keep class com.artemzin.qualitymatters.api.entities.** { *; }
```

Comment:
AutoValue does not require any proguard rules so I think we can get rid of this.